MomsSpaghetti/Pages/Index.cshtml.cs

```diff
@@ -257,48 +259,65 @@ public ActionResult OnPostDeleteLiked()
         }
 
 
-    
-        public async Task<IActionResult>  OnPost()
-        {   
-            String responseContent="";
+
+        public async Task<IActionResult> OnPost()
+        {
+            String responseContent = "";
             var responseObject = new { };
+            if (SearchRecipes.Query == null)
```

Comment:
If the query is null then the page gets returned .
Source: https://michaelscodingspot.com/7-debugging-techniques-know-c-net/

Reply:
Which page?